internal/sinks/alertmanager.go

```diff
@@ -84,7 +84,8 @@ func (s *AlertmanagerSink) Configure(c Client, config map[string][]byte) error {
 	if ok {
 		caCertPool, err = x509.SystemCertPool()
 		if err != nil {
-			return errors.Wrap(err, "invalid Alertmanager config: failed to get system cert pool")
+			s.log.Error(err, "failed to get system cert pool; using empty pool")
+			caCertPool = x509.NewCertPool()
```

Comment:
This change might result in nil pointer derefs. If `x509.SystemCertPool` returns an error, there's a chance `caCertPool` in our code is nil. See https://github.com/golang/go/blob/53270be21c5e14a2d15ada2622f20503d97cb89e/src/crypto/x509/root_unix.go#L81, which is a helper it calls.

Reply:
lgtm